index.js

```diff
@@ -0,0 +1,46 @@
+const bodyParser = require('body-parser')
+const express = require('express')
+const Movies = require('./models')
+
+
+const app = express()
+
+app.get('/movies', async (request, response) => {
+  const movies = await models.Movies.findAll({
+    where: { directors: request.params.title }
+  })
+  
+  return movies.length
+    ? response.send(movies)
+    : response.sendStatus(404)
+})
+
+app.get('/movies/:title', async(request, response) => {
+  
+  const matchingmovies = await Movies.findAll({
```

Comment:
You imported Movies from your models folder. This means that Movies is an object with a parameter Movies that is the model. This would have to be `Movies.Movies.findAll` based on how you named things.